### ISSUE TITLE
GUI error. The panel is of the wrong size.

### ISSUE THREAD

**Issue by LeoKudrik** [BBasile/Coedit#251]
**aur coedit-bin 3.6.3-1**
The problem occurs immediately after installation. Any information you provide.

![screenshot_2018-02-21_23-13-18](https://user-images.githubusercontent.com/362985/36504049-4290f580-1760-11e8-8caf-847fa90f6c82.png)
```
$ uname -a
Linux leok-thinkpad 4.14.20-2-MANJARO #1 SMP PREEMPT Sat Feb 17 21:47:27 CET 2018 x86_64 GNU/Linux
$ xfce4-about -V
xfce4-about 4.12.1 (Xfce 4.12)
```
Thanks for coedit!

**Comment by BBasile** [comment]
You can use CTRL + drag on the top splitter, just over the caption "Source Editor"

**Comment by BBasile** [comment]
I close but it's not fixed, opened #252 for this problem.
